examples/webpack2/src/main.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import { render } from 'react-dom'
+import { Button, Container, Header } from 'semantic-ui-react'
```

Comment:
Hm, seems I'll have to put in more due diligence to investigate here.  I will do this as soon as I can.